ERROR - 2023-01-05 12:04:00 --> Error connecting to the database: mysqli_sql_exception: Connection timed out in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(171): mysqli->real_connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#6 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#7 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection timed out in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#3 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#5 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#6 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#7 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#8 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-01-05 12:04:00 --> Unable to connect to the database.
Main connection [MySQLi]: Connection timed out
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(586): CodeIgniter\Model->doFindAll()
 5 APPPATH/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
 6 APPPATH/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
 7 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 8 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 9 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
10 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:12:33 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(171): mysqli->real_connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#6 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#7 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#3 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#5 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#6 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#7 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#8 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-01-05 12:12:33 --> Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(586): CodeIgniter\Model->doFindAll()
 5 APPPATH/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
 6 APPPATH/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
 7 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 8 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 9 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
10 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:12:35 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(171): mysqli->real_connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#6 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#7 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /var/www/html/sgapi/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#3 /var/www/html/sgapi/system/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/sgapi/system/BaseModel.php(586): CodeIgniter\Model->doFindAll()
#5 /var/www/html/sgapi/app/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
#6 /var/www/html/sgapi/app/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
#7 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#8 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#9 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#10 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-01-05 12:12:35 --> Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH/Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(586): CodeIgniter\Model->doFindAll()
 5 APPPATH/Models/AtendimentoModel.php(46): CodeIgniter\BaseModel->findAll()
 6 APPPATH/Controllers/Api.php(15): App\Models\AtendimentoModel->getcontador()
 7 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 8 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 9 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
10 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:23:59 --> There is no data to update.
in SYSTEMPATH/BaseModel.php on line 1600.
 1 SYSTEMPATH/BaseModel.php(1600): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset()
 2 SYSTEMPATH/BaseModel.php(854): CodeIgniter\BaseModel->transformDataToArray()
 3 SYSTEMPATH/Model.php(721): CodeIgniter\BaseModel->update()
 4 APPPATH/Models/AtendimentoModel.php(55): CodeIgniter\Model->update()
 5 APPPATH/Controllers/Api.php(17): App\Models\AtendimentoModel->updatecontador()
 6 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 7 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:30:13 --> Class 'app\Models\ServicosModel' not found
in APPPATH/Controllers/Api.php on line 14.
 1 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 2 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 3 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 4 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:31:37 --> Array to string conversion
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:32:09 --> Undefined offset: 0
in APPPATH/Models/ServicosModel.php on line 50.
 1 APPPATH/Models/ServicosModel.php(50): CodeIgniter\Debug\Exceptions->errorHandler()
 2 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 3 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 4 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:32:19 --> Undefined index: sigla
in APPPATH/Models/ServicosModel.php on line 50.
 1 APPPATH/Models/ServicosModel.php(50): CodeIgniter\Debug\Exceptions->errorHandler()
 2 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 3 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 4 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:32:41 --> Undefined offset: 0
in APPPATH/Models/ServicosModel.php on line 50.
 1 APPPATH/Models/ServicosModel.php(50): CodeIgniter\Debug\Exceptions->errorHandler()
 2 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 3 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 4 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:37:02 --> Argument 1 passed to CodeIgniter\BaseModel::findAll() must be of the type int, string given, called in /var/www/html/sgapi/app/Models/ServicosModel.php on line 49
in SYSTEMPATH/BaseModel.php on line 569.
 1 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->findAll()
 2 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 3 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 4 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:37:34 --> mysqli_sql_exception: Unknown column 'servicos_unidades.servicos_id' in 'where clause' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(525): CodeIgniter\Model->doFind()
#6 /var/www/html/sgapi/app/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
#7 /var/www/html/sgapi/app/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-01-05 12:37:34 --> Unknown column 'servicos_unidades.servicos_id' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(525): CodeIgniter\Model->doFind()
 7 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
 8 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 9 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
10 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
11 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:37:57 --> mysqli_sql_exception: Unknown column 'servicos_unidades.servicos_id' in 'where clause' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(525): CodeIgniter\Model->doFind()
#6 /var/www/html/sgapi/app/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
#7 /var/www/html/sgapi/app/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-01-05 12:37:57 --> Unknown column 'servicos_unidades.servicos_id' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(525): CodeIgniter\Model->doFind()
 7 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
 8 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 9 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
10 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
11 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:38:01 --> mysqli_sql_exception: Unknown column 'servicos_unidades.servicos_id' in 'where clause' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(525): CodeIgniter\Model->doFind()
#6 /var/www/html/sgapi/app/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
#7 /var/www/html/sgapi/app/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-01-05 12:38:01 --> Unknown column 'servicos_unidades.servicos_id' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(525): CodeIgniter\Model->doFind()
 7 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
 8 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 9 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
10 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
11 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:38:13 --> mysqli_sql_exception: Unknown column 'servicos_unidades.servicos_id' in 'where clause' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(525): CodeIgniter\Model->doFind()
#6 /var/www/html/sgapi/app/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
#7 /var/www/html/sgapi/app/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-01-05 12:38:13 --> Unknown column 'servicos_unidades.servicos_id' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(525): CodeIgniter\Model->doFind()
 7 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
 8 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 9 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
10 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
11 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 12:38:18 --> mysqli_sql_exception: Unknown column 'servicos_unidades.servicos_id' in 'where clause' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/sgapi/system/BaseModel.php(525): CodeIgniter\Model->doFind()
#6 /var/www/html/sgapi/app/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
#7 /var/www/html/sgapi/app/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
#8 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#10 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#11 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-01-05 12:38:18 --> Unknown column 'servicos_unidades.servicos_id' in 'where clause'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(199): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/BaseModel.php(525): CodeIgniter\Model->doFind()
 7 APPPATH/Models/ServicosModel.php(49): CodeIgniter\BaseModel->find()
 8 APPPATH/Controllers/Api.php(20): App\Models\ServicosModel->siglaservico()
 9 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
10 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
11 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
12 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:57:00 --> Undefined index: sigla
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:58:46 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:58:58 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:59:00 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:59:01 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 12:59:46 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:00:06 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:00:21 --> Undefined offset: 0
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:03:06 --> Illegal string offset 'sigla'
in APPPATH/Controllers/Api.php on line 22.
 1 APPPATH/Controllers/Api.php(22): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:33:02 --> Constant expression contains invalid operations
in APPPATH/Models/SenhaModel.php on line 49.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-01-05 13:33:54 --> Too few arguments to function App\Models\SenhaModel::novasenha(), 2 passed in /var/www/html/sgapi/app/Controllers/Api.php on line 26 and exactly 4 expected
in APPPATH/Models/SenhaModel.php on line 49.
 1 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
 2 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 3 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:39:30 --> Allowed fields must be specified for model: App\Models\SenhaModel
in SYSTEMPATH/BaseModel.php on line 1162.
 1 SYSTEMPATH/BaseModel.php(1162): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields()
 2 SYSTEMPATH/BaseModel.php(725): CodeIgniter\BaseModel->doProtectFields()
 3 SYSTEMPATH/Model.php(695): CodeIgniter\BaseModel->insert()
 4 APPPATH/Models/SenhaModel.php(58): CodeIgniter\Model->insert()
 5 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
 6 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 7 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-05 13:40:51 --> Allowed fields must be specified for model: App\Models\SenhaModel
in SYSTEMPATH/BaseModel.php on line 1162.
 1 SYSTEMPATH/BaseModel.php(1162): CodeIgniter\Database\Exceptions\DataException::forInvalidAllowedFields()
 2 SYSTEMPATH/BaseModel.php(725): CodeIgniter\BaseModel->doProtectFields()
 3 SYSTEMPATH/Model.php(695): CodeIgniter\BaseModel->insert()
 4 APPPATH/Models/SenhaModel.php(59): CodeIgniter\Model->insert()
 5 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
 6 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
 7 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
 8 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 13:41:43 --> mysqli_sql_exception: Table 'novosga_db.atendimento' doesn't exist in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/sgapi/system/BaseModel.php(750): CodeIgniter\Model->doInsert()
#6 /var/www/html/sgapi/system/Model.php(695): CodeIgniter\BaseModel->insert()
#7 /var/www/html/sgapi/app/Models/SenhaModel.php(60): CodeIgniter\Model->insert()
#8 /var/www/html/sgapi/app/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#10 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#11 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#12 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-05 13:41:43 --> Table 'novosga_db.atendimento' doesn't exist
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(750): CodeIgniter\Model->doInsert()
 7 SYSTEMPATH/Model.php(695): CodeIgniter\BaseModel->insert()
 8 APPPATH/Models/SenhaModel.php(60): CodeIgniter\Model->insert()
 9 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
10 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
11 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
12 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
13 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 13:52:12 --> mysqli_sql_exception: Unknown column 'prioridade' in 'field list' in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/sgapi/system/BaseModel.php(750): CodeIgniter\Model->doInsert()
#6 /var/www/html/sgapi/system/Model.php(695): CodeIgniter\BaseModel->insert()
#7 /var/www/html/sgapi/app/Models/SenhaModel.php(63): CodeIgniter\Model->insert()
#8 /var/www/html/sgapi/app/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#10 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#11 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#12 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-05 13:52:12 --> Unknown column 'prioridade' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(750): CodeIgniter\Model->doInsert()
 7 SYSTEMPATH/Model.php(695): CodeIgniter\BaseModel->insert()
 8 APPPATH/Models/SenhaModel.php(63): CodeIgniter\Model->insert()
 9 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
10 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
11 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
12 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
13 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-05 13:52:34 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`novosga_db`.`atendimentos`, CONSTRAINT `FK_29E906E7EDF4B99B` FOREIGN KEY (`unidade_id`) REFERENCES `unidades` (`id`)) in /var/www/html/sgapi/system/Database/MySQLi/Connection.php:292
Stack trace:
#0 /var/www/html/sgapi/system/Database/MySQLi/Connection.php(292): mysqli->query()
#1 /var/www/html/sgapi/system/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
#2 /var/www/html/sgapi/system/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
#3 /var/www/html/sgapi/system/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
#4 /var/www/html/sgapi/system/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
#5 /var/www/html/sgapi/system/BaseModel.php(750): CodeIgniter\Model->doInsert()
#6 /var/www/html/sgapi/system/Model.php(695): CodeIgniter\BaseModel->insert()
#7 /var/www/html/sgapi/app/Models/SenhaModel.php(63): CodeIgniter\Model->insert()
#8 /var/www/html/sgapi/app/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
#9 /var/www/html/sgapi/system/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
#10 /var/www/html/sgapi/system/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
#11 /var/www/html/sgapi/system/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
#12 /var/www/html/sgapi/index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-05 13:52:34 --> Cannot add or update a child row: a foreign key constraint fails (`novosga_db`.`atendimentos`, CONSTRAINT `FK_29E906E7EDF4B99B` FOREIGN KEY (`unidade_id`) REFERENCES `unidades` (`id`))
in SYSTEMPATH/Database/MySQLi/Connection.php on line 292.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(292): mysqli->query()
 2 SYSTEMPATH/Database/BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute()
 3 SYSTEMPATH/Database/BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery()
 4 SYSTEMPATH/Database/BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query()
 5 SYSTEMPATH/Model.php(303): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH/BaseModel.php(750): CodeIgniter\Model->doInsert()
 7 SYSTEMPATH/Model.php(695): CodeIgniter\BaseModel->insert()
 8 APPPATH/Models/SenhaModel.php(63): CodeIgniter\Model->insert()
 9 APPPATH/Controllers/Api.php(26): App\Models\SenhaModel->novasenha()
10 SYSTEMPATH/CodeIgniter.php(925): App\Controllers\Api->tirarsenha()
11 SYSTEMPATH/CodeIgniter.php(480): CodeIgniter\CodeIgniter->runController()
12 SYSTEMPATH/CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest()
13 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
